rental.rb

```diff
@@ -4,8 +4,8 @@ class Rental
   def initialize(date, person, book)
     @date = date
     @person = person
-    person.rental << self
+    person.rentals << self
     @book = book
-    book.rental << self
+    book.rentals << self
   end
 end
```

Comment:
- Hey, when I try renting a book, I get the error below. Please, try fix it so that one can rent a book as this a core functionality of the app.

![Screenshot from 2022-11-16 21-59-02](https://user-images.githubusercontent.com/59397159/202293348-58b45177-ec23-4a18-b19d-d4e185a8e6e0.png)

Reply:
@zieeco  hi but it work properly for me

Reply:
@zieeco  can you please recheck?

Reply:
this the result 
<img width="379" alt="Screenshot 2022-11-16 232905" src="https://user-images.githubusercontent.com/103565978/202298874-8dc08b6e-0023-41e4-8f5a-0ca1322b5cb1.png">
<img width="335" alt="Screenshot 2022-11-16 233005" src="https://user-images.githubusercontent.com/103565978/202298908-f4f75449-3167-4065-805f-47a069385cd5.png">
<img width="394" alt="Screenshot 2022-11-16 232945" src="https://user-images.githubusercontent.com/103565978/202298948-73cb05f9-2ad5-42cb-ab14-a006363a572f.png">

<img width="299" alt="Screenshot 2022-11-16 233033" src="https://user-images.githubusercontent.com/103565978/202298971-c6417524-13f2-4f2c-a8a8-f7d0b2adf137.png">

Reply:
@X-Elie-X 

- I did but it was throwing that error, however, if it works fine now, kindly request for another review. 
Thanks.